### DISCUSSION TITLE
Proposta do carrossel para divulgação do projeto

### DISCUSSION THREAD

**Discussion by mariimmj** [hywenklis/buddy-api#46] [Mostra e diz]
## Introdução

Olá pessoal!

Meu nome é Maria e estou a algum tempo estudando sobre designer e social media. Estou muito empolgada com o projeto Buddy e acredito que ele tem um enorme potencial para fazer a diferença na vida dos animais resgatados. Para contribuir com o projeto, criei um carrossel de design que pode ser usado para promover a iniciativa e atrair mais colaboradores.

## O Carrossel de Design

O carrossel é composto por 7 slides, cada um destacando aspectos importantes do projeto Buddy, como:

1. **O que é o Buddy?**: Explicando o que é o Buddy e seu objetivo.
2. **Importância da Adoção de Animais**: Por que é importante adotar e como o Buddy facilita esse processo.
3. **Como Contribuir (público geral)**: Informações sobre como o público pode contribuir. 
4. **Como Contribuir (público mais técnico)**: Informações sobre como desenvolvedores, designers e outros podem se juntar ao projeto.
5. **Chamada para Ação**: Incentivando a participação do público para a causa.
6. **Chamada para acessar o site**:  Incentivando a visita ao site e a participação na comunidade.
7. **Apresentação dos autores**: Apresentando o autor de cada projeto (Frontend e Backend)

Link do design criado no app Canva: https://www.canva.com/design/DAGLOf8KWQk/WHmBbjhekg8yeDMHxUaI8Q/edit?utm_content=DAGLOf8KWQk&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton
## Como Contribuir

Gostaria de compartilhar o carrossel e discutir como ele pode ser implementado nas mídias sociais. Acredito que isso pode ajudar a aumentar a visibilidade do projeto e atrair mais contribuições. Estou aberta feedback e sugestões para melhorar o design. 🙂

## Próximos Passos

- **Revisão do Carrossel**: Peço que revisem o carrossel e compartilhem suas opiniões.
- **Implementação nas Redes Sociais**: Planejamento de uma estratégia de postagem usando o carrossel.
- **Colaboração Contínua**: Estou disposta a continuar contribuindo com designs e estratégias de mídia social para o Buddy.

Peço permissão para que eu possa compartilhar as imagens nas minhas redes sociais marcando os autores do projeto. Agradeço a todos pela atenção e estou ansiosa para colaborar com esta incrível iniciativa! 🫱🏻‍🫲🏼

![1](https://github.com/user-attachments/assets/bab031b1-028a-4a6e-b28a-3037443b4ff7)
![2](https://github.com/user-attachments/assets/c9f31344-ea3e-4d6f-9f1e-97e58e6d87c3)
![3](https://github.com/user-attachments/assets/091209af-9b96-4cf8-9a06-918e6514de07)
![4](https://github.com/user-attachments/assets/ff934005-40b6-44a3-81b6-9ab33975fafb)
![5](https://github.com/user-attachments/assets/8e8fe4c5-af5f-4afc-b813-9c16b45f9be7)
![6](https://github.com/user-attachments/assets/ce5d66aa-3fb6-4949-8441-a4ba1584bf53)
![7](https://github.com/user-attachments/assets/ba46fbc2-41e9-42d2-9011-027e57321d94)
![8](https://github.com/user-attachments/assets/d052b6fa-0370-408e-9b3d-dea3fa033f39)

**Comment by genesluna** [comment]
Olá, Maria. Gostei muito do seu trabalho. :heart: A contribuição é de grande valia para o nosso projeto. Da minha parte, você tem toda permissão para compartilhar as imagens. Muito obrigado por sua colaboração.

**Reply by mariimmj** [reply]
Eu que sou grata. Vocês estão de parabéns! 😁

**Comment by hywenklis** [comment]
Muito obrigado @mariimmj  por contribuir. Da minha parte tem total permissão para divulgação das imagens. Ficou excelente seu trabalho ❤️

**Reply by mariimmj** [reply]
Eu que agradeço, o trabalho de vocês foi espetacular!❤️😁
